cluster_migration_core/cluster_migration_core/clients/rest_client_default.py

```diff
@@ -41,8 +41,18 @@ def get_doc_by_id(self, port: int, index: str, doc_id: str) -> ops.RESTResponse:
         """
         rest_path = ops.RESTPath(port=port, suffix=f"{index}/_doc/{doc_id}")
         params = {"pretty": "true"}
+
         return ops.perform_get(rest_path=rest_path, params=params)
 
+    def create_an_index(self, port: int, index: str) -> ops.RESTResponse:
+        """
+        Creates an index
+        """
+        rest_path = ops.RESTPath(port=port, suffix=f"{index}")
```

Comment:
As a tip I don't think you necessarily need the f string `f"{index}"` here as index is already a string and is not joined with anything, `index` by itself should work as well

Reply:
Ah I see, thanks! I'll address this.